serum-390/soen390-team16-godzilla
# Shipping frontend

# Description
This PR creates the shipping management page that allows scheduling and canceling order shippings

## Link to work item

TODO: Include a link to the work item E.g.: #1

## How to Test
navigate to shipping page , click on schedule shipping , enter the order id , the date , and the shipping method. then submit. the alert displays the feedback received from backend for processing this request, if the feedback is successful, you can refresh the page and see the new shipping item with status = scheduled. you can also click on the cancel button to cancel shipping
